CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Given a version number MAJOR.MINOR.PATCH, increment:
 
 
 ## [Unreleased]
+### Added
+- Reversed Deposit pdf
```

Comment:
Uma sugestão para o CHANGELOG:
function to deposit.log resource get Deposit reversal receipt

Coloquei só algo mais específico pro usuário bater o olho no CHANGELOG e ja entender do que se trata

Reply:
Boooa!
Vou ajustar aqui